examples/MultipleRedeemers.purs

```diff
@@ -0,0 +1,254 @@
+module Examples.MultipleRedeemers (threeRedeemerContract) where
+
+import Prelude
```

Comment:
I'd recommend adjusting the imports. Having these examples is partially to show how we intend the library to be used. Our goal is that users only need to import `Contract.Prelude` as a replacement for `Prelude` and then don't need to worry about things like `Maybe`, `Either`, `(/\)`, etc...

Also there are several imports of internal modules from `Types.*`, etc... For example, `TokenName` should be imported from `Contract.Value`